src/browser.js

```diff
@@ -67,51 +69,48 @@ export class FetchError {
 }
 
 // Returns a promise that will resolve to the content of the given path.
-export function fetchUrl(path, params, headers) {
-  return new Promise(function(resolve, reject) {
-    var xhr = new XMLHttpRequest();
-    if (params) {
-      var separator = '?';
-      params.forEach(function(param) {
-        var key = encodeURIComponent(String(param[0]));
-        var val = encodeURIComponent(String(param[1]));
-        path += separator + key + '=' + val;
-        separator = '&';
-      });
-    }
-    xhr.open('GET', path, true);
-    if (headers) {
-      utils.Map.wrap(headers).forEach(function(key, value) {
-        xhr.setRequestHeader(key, value);
-      });
-    }
-    xhr.onreadystatechange = function() {
-      if (xhr.readyState != 4)
-        return;
+export function fetchUrl(path, params, headers = {}) {
```

Comment:
Can you change the caller in gerrit.js to add `'mode': 'no-cors'` to the `headers` parameter?

Reply:
Nevermind. The problem is that I forgot to use the correct extension id to get the allowance to use CORS.